test/distributed/checkpoint/fsdp/test_fsdp_dsd.py

```diff
@@ -521,7 +519,7 @@ def _get_base_model(mlp_dim: int = 2):
                 fsdp2_tp_optim = torch.optim.AdamW(fsdp2_tp_model.parameters(), lr=0.1)
 
                 # one-step training to modify state dict
-                inp = torch.randn((2,), device=self.rank)
+                inp = torch.randn((mlp_dim,), device=self.rank)
```

Comment:
I see, so the test was only testing the case `mlp_dim=2`? Shall we change the line above `base_model = _get_base_model().cuda()` to `base_model = _get_base_model(mlp_dim).cuda()`, where `mlp_dim` is a variable? Right now it's fixed `mlp_dim=5`. 

It doesn't have to be in this PR, you can add a TODO